chapters/en/chapter7/5.mdx

```diff
@@ -289,9 +289,10 @@ def preprocess_function(examples):
         max_length=max_input_length,
         truncation=True,
     )
-    labels = tokenizer(text_target=targets, max_length=max_target_length, truncation=True)
+    labels = tokenizer(
+        examples["review_title"], max_length=max_target_length, truncation=True
+    )
```

Comment:
`targets` wasn't defined, so I think this code just didn't work before? Not sure how it got like this.

Reply:
![image](https://user-images.githubusercontent.com/26859204/197225921-b873b3d9-4ebe-40f2-9f3b-937e4c0304b8.png)

Reply:
That code was used for PyTorch too!

Reply:
I confirm, I had to modify this line in the French adaptation for it to work